wewatch-client/src/components/Lobby/Lobby.js

```diff
@@ -78,40 +80,55 @@ const Lobby = ({location}) => {
 
     useEffect(() => {
         // set boolean for redirecting to swipe screen to be true, renders redirect component
-        socket.on('sessionMembers', ({roomId, users, host, top10}) => {
+        socket.on('sessionMembers', ({roomId, users, host}) => {
```

Comment:
Do we no longer need to pass the `top10` movies object as an argument with the event?